jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -635,6 +635,7 @@ void java_bytecode_convert_classt::convert(
     symbolt new_symbol;
 
     new_symbol.is_static_lifetime=true;
+    new_symbol.is_static_nondet_init=true;
```

Comment:
The commit message could be more informative (e.g. Add is_static_nondet_init to symbol)